tryffle/uploaded_pdf/templates/pages.html

```diff
@@ -0,0 +1,16 @@
+<html>
+    <head><title>Merchex</title></head>
+    <body>
+    {% block content %}
+
+<h1>Pages du document : {{ document.title }}</h1>
+
+<ul>
+ {% for page in pages %}
```

Comment:
Suite du commentaire plus bas, tu peux remplacer `pages` par `document.page_set`.

{% for page in document.page_set %}